pegasus/sites.v3/code.org/views/maker_devices.haml

```diff
@@ -0,0 +1,13 @@
+.action-block__wrapper.action-block__wrapper--two-col
+  .action-block.action-block--one-col
+    %h3 Circuit Playground
+    %img{src: "/images/maker/maker-device-circuit-playground.png", alt: ""}
+    %a.link-button{href: curriculum_link_cp, aria:{label: aria_label_cp}}
```

Comment:
Where is `aria_label_cp` defined? I see it elsewhere in the repo, but not sure how it's getting through to here.

Reply:
Oh ignore me! Just looked more closely about how it's getting passed in –– looks great!